app/components/post_list/post/body/content/permalink_preview/permalink_preview.tsx

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useMemo, useCallback, useEffect} from 'react';
+import {Text, View, Pressable} from 'react-native';
+
+import {fetchUsersByIds} from '@actions/remote/user';
+import EditedIndicator from '@components/EditedIndicator';
```

Comment:
EditedIndicator (the path) should be `edited_indicator` but ok to change in another PR

Reply:
Ah, yes. I will update it in this PR.